test-harness/helpers.ts

```diff
@@ -1,3 +1,31 @@
+import { Dictionary } from '@stoplight/types/dist';
+import * as xmlDiff from 'diff-js-xml';
+import * as parser from 'fast-xml-parser';
+import { get, omit } from 'lodash';
+
+type Result = { body: string; headers: Dictionary<string, string> };
+
+const xmlValidator = {
+  test: (contentType: string, content: string) => {
+    const doesContentTypeMatch = !!contentType.match(/application\/.*xml/);
```

Comment:
Did you know, `text/xml` is equally correct to describe XML. An interesting chat on the topic: https://stackoverflow.com/questions/4832357/whats-the-difference-between-text-xml-vs-application-xml-for-webservice-respons

Reply:
Out of curiosity, can package like https://www.npmjs.com/package/accepts help with this type of stuff? There must be a package that checks `package.isMediaType(content, 'xml')`.

Reply:
Yep that's exactly the sort of thing accepts is designed to handle, and it is used for this elsewhere in Prism.

I'd rather step back a level, and find out why we are even trying to detect XML and do something special for it. I don't understand the premise. https://github.com/stoplightio/prism/pull/576#issuecomment-527118433